bc_obps/common/tests/endpoints/v1/dashboard/test_dashboard_authorization.py

```diff
@@ -0,0 +1,18 @@
+from registration.tests.utils.helpers import CommonTestSetup
+from registration.tests.utils.helpers import TestUtils
+from common.constants import BASE_ENDPOINT
+
+# make pythontests ARGS='common/tests/endpoints/v1/dashboard/test_dashboard_authorization.py' 
+class TestDashboardDataEndpointAuthorization(CommonTestSetup):
```

Comment:
I suggest adding another test to ensure we get the right data based on different roles and dashboard parameters.

Reply:
@Sepehr-Sobhani as mentioned in "Notes", my thinking is "Limited python tests to authorization assuming test of dashboard specific JSON will be tested in e2e tests" but, happy to add here if you think it advisable.

Reply:
Sorry for not paying enough attention to your notes. Of course, we can ignore that. I always assume that I don't know what is happening on the front-end side, so I try to approach everything in a way that ensures the backend is independent. This way, we aren't reliant on tests from the other side.